### ISSUE TITLE
Option to specify class on preview's html

### ISSUE THREAD

**Issue by dricholm** [hipstersmoothie/storybook-dark-mode#133]
Hello,
Thanks for this plugin, it works great!

There is one extra feature I'd like to have: add a dark class to the preview's <html> element.
Seems like this was requested in #101 and implemented in #102, but the implementation adds the class to the <body> element in the preview iframe.
The reason I want it for <html> is so body can also apply dark styles to itself if a parent element has `.dark`. In my case I use it with Tailwind CSS, where the dark mode will be applied if a parent element has the class `.dark` applied. See [docs](https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually) for more information about this.

Here are two pictures to better demonstrate this. This is the current state, when the class is applied to the body:
![image](https://user-images.githubusercontent.com/32329112/104182859-5e684100-5411-11eb-95f3-e756b75be345.png)
I added `class="dark"` in the developer tools to the preview <html> element:
![image](https://user-images.githubusercontent.com/32329112/104182906-7213a780-5411-11eb-807c-f6af32fbf974.png)
As you can see, on the top it will only apply the dark theme to the component, while on the bottom it will also be applied to the body.

I tried it out by changing the [line](https://github.com/hipstersmoothie/storybook-dark-mode/blob/master/src/Tool.tsx#L70) about body to select the html instead and it worked locally.
What do you think about this?
Could this be added? If yes, I can help with a PR if it's okay. In that case what should be the default: adding it to HTML or body. Or add a new option to specify this or other possibilities?

**Comment by lharress** [comment]
Had this problem as well, may not be the best solution, but I just used a global decorator;

```javascript
(Story) => {
    const isDarkMode = useDarkMode();

    useEffect(() => {
        if (!isDarkMode) {
            return;
        }

        document.documentElement.classList.add("dark");

        return () => {
            document.documentElement.classList.remove("dark");
        };
    }, [isDarkMode]);

    return <Story />;
},
```

**Comment by hipstersmoothie** [comment]
I'd be open to merging a PR for this. The option could be name `classTarget` and default to `body`.

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v1.0.7`](https://github.com/hipstersmoothie/storybook-dark-mode/releases/tag/v1.0.7) :rocket:
